src/containers/loan/components/LoanList.js

```diff
@@ -10,14 +10,16 @@ export default function LoanList(props) {
     const listItems =
         loans &&
         loans
-            .reverse()
             .filter(props.filter ? props.filter : loan => loan.isRepayable)
             .map(loan => (
                 <MyListGroup.Row key={`loan-${loan.id}`}>
                     <LoanListDetails loan={loan} />
                     {props.selectComponent && <props.selectComponent loanId={loan.id} />}
                 </MyListGroup.Row>
-            ));
+            ))
+            .sort((a, b) => {
```

Comment:
Put the 'sort' fn above the 'map'. (because this a.props.children[0].props.loan.maturity is ugly)
(of course, everywhere in this PR)